.github/workflows/main-pr.yml

```
@@ -158,3 +158,35 @@ jobs:
+    name: Build and Publish (Faucet)
+    needs: test-nitronode
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v6
+
+      - name: Log in to GitHub Container Registry
+        uses: docker/login-action@v2
```

actionlint (1.7.12)

[error] the runner of "docker/login-action@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)